tests/core/htmlparser/elementfindone.js

```diff
@@ -0,0 +1,44 @@
+/* bender-tags: editor */
+
+( function() {
+	'use strict';
+
+	var testElement = CKEDITOR.htmlParser.fragment.fromHtml(
```

Comment:
I wonder if it wouldn't be more readable to move the HTML into separate file, `elementfindone.html`.